verilog/parser/verilog.y

```diff
@@ -7037,10 +7037,14 @@ bind_target_instance
   ;
 bind_instantiation
   /* similar to block_item_decl instantiation */
-  : class_id gate_instance_or_register_variable_list
+  : bind_class_id gate_instance_or_register_variable_list
     { $$ = MakeTaggedNode(N::kBindTargetInstance, $1, $2); }
```

Comment:
Alternatively, you could just write:

```
{ $$ = MakeTaggedNode(N::kBindTargetInstance,
                      MakeTaggedNode(N::kInstantiationType, $1),
                      $2);
}
```

without adding a single-use nonterminal.